openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/workflows/metadata.md

```diff
@@ -0,0 +1,62 @@
+# Metadata
+
+DashboardService Metadata Pipeline Configuration.
+
+## Properties
+
+
+
+$$section
+
+### Dashboard Filter Pattern $(id="dashboardFilterPattern")
+
+Dashboard filter patterns to control whether or not to include dashboard as part of metadata ingestion. Enter the regex pattern form the for including or excluding the dashboard.
```

Comment:
should we add a link to the docs? even if its for the databases, the examples should work here too. WE could have a simple example in here as well